CODEOWNERS

```diff
@@ -0,0 +1,146 @@
+# This file contains the list of maintainers (i.e. people who own the
+# packages and can commit changes) for various parts of edk2-platforms.
+
+# EDK II Platforms maintainers
+# ============================
+* @leiflindholm @mdkinney
+
+# EDK II Platforms Packages
+# =========================
+
+/Platform/96Boards/** @ardbiesheuvel @leiflindholm
+
+/Platform/AMD/** @changab @abdattar @pbgrimes
+
+/Platform/AMD/AgesaModulePkg/** @changab @abdattar @pbgrimes
+/Platform/AMD/AgesaPkg/**       @changab @abdattar @pbgrimes
+/Platform/AMD/AmdCbsPkg/**      @changab @abdattar @pbgrimes
+/Platform/AMD/AmdCpmPkg/**      @changab @abdattar @pbgrimes
+
+# AMD Seattle platforms
+/Platform/AMD/OverdriveBoard/** @changab @abdattar @pbgrimes @ardbiesheuvel @leiflindholm
+/Silicon/AMD/Styx/**            @changab @abdattar @pbgrimes @ardbiesheuvel @leiflindholm
+/Platform/LeMaker/CelloBoard/** @ardbiesheuvel @leiflindholm
+/Platform/SoftIron/             @ardbiesheuvel @leiflindholm
+
+# AMD VanGoghBoard
+/Platform/AMD/VanGoghBoard/** @changab @abdattar @pbgrimes @exinghr
+
+# Ampere Computing
+/Platform/Ampere/** @nhivp
+/Silicon/Ampere/** @nhivp
+
+# ARM
+/Platform/ARM/** @samimujawar
+/Silicon/ARM/** @samimujawar
+
+# BeagleBoard:
+/Platform/BeagleBoard/** @ardbiesheuvel @leiflindholm
+/Silicon/TexasInstruments/** @ardbiesheuvel @leiflindholm
+
+# DisplayLink
+/Drivers/DisplayLink/** @leiflindholm @ardbiesheuvel

```

Comment:
Please make it alphabetical like in other places where both Ard and Leif are listed.

Reply:
Good point.